thirdparty/github.go

```diff
@@ -1881,3 +1882,67 @@ func GetCheckRun(ctx context.Context, owner, repo string, checkRunID int64) (*gi
 	}
 	return checkRun, nil
 }
+// Returns an empty string if no co-author information is found.
+func ExtractCoAuthorEmail(message string) string {
+	re := regexp.MustCompile(`(?i)Co-Authored-By:.*<([^>]+)>`)
+	matches := re.FindStringSubmatch(message)
+	if len(matches) > 1 {
+		return matches[1]
+	}
+	return ""
+}
+
+func GetCoAuthorEmail(ctx context.Context, owner, repo string, prNumber int) (string, error) {
```

Comment:
this should have a function comment (always public functions should have a function comment)